tests/rpc_test.go

```diff
@@ -73,11 +71,33 @@ func TestMain(m *testing.M) {
 		HOST = "http://localhost:8545"
 	}
 
+	var err error
+	from, err = getAddress()
+	if err != nil {
+		fmt.Printf("failed to get account: %s\n", err)
+		os.Exit(1)
```

Comment:
use `Fatal` or `Fatalf`?

Reply:
you mean `t.Fatal`? there's no `testing.T` available in this func since it's the `TestMain`